src/helpers/daily-data-providers/apple-health-sleep.ts

```diff
@@ -85,7 +92,7 @@ function coreSleep(startDate: Date, endDate: Date, value: "Asleep" | "InBed") {
 
 		ddp.forEach((d) => {
 			if (!d.observationDate || !d.startDate) { return; }
-			if (d.value != value) { return; }
+			if (!value.find(x => x /* as string */ == d.value) ) { return; }
```

Comment:
remove comment (it's unnecessary right?)

Reply:
Thought I might have some issues with typescript, but it is not complaining